libs/libc/misc/lib_mutex.c

```diff
@@ -35,30 +35,14 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#define NXMUTEX_RESET          ((pid_t)-2)
-
 /****************************************************************************
  * Private Functions
```

Comment:
remove

Reply:
remove what? NXMUTEX_RESET is already removed I believe.